runtimes/polimec/src/lib.rs

```diff
@@ -1195,17 +1221,27 @@ impl_runtime_apis! {
 			use cumulus_pallet_session_benchmarking::Pallet as SessionBench;
 			use xcm::latest::prelude::*;
 			use pallet_xcm::benchmarking::Pallet as PalletXcmExtrinsiscsBenchmark;
-			/// TODO: Update these benchmarks once we enable PLMC Teleportation
+			/// TODO: Update these janky implementations once we upgrade pallet_xcm. The jankyness is due to badly written pallet-xcm code
```

Comment:
Can you keep the old TODO as well? And maybe remove the comment on badly written as its an open source repo 🤣